.buildkite/pipeline.yml

```diff
@@ -44,6 +44,27 @@ steps:
     agents:
       queue: "scala"
 
+  - label: "Test concepts API"
+    key: "concepts-test"
+    plugins:
+      - docker-compose#v3.9.0:
+          run: concepts
+          command: ["yarn", "test"]
+
+  - label: "Publish concepts API (ECR)"
```

Comment:
I realise this is probably copied from the dotorg repo; could we use consistent labelling with the other tasks in this repo please?

Reply:
It wasn't - that's not a straightforward thing to do as (1) the other tasks handle the test/push logic inside `run_job.py` and (2) they're very inconsistently named as it is. I've renamed some things a little but there is no way of doing this consistently with the other APIs as the steps aren't exactly analogous.